packages/frontend/src/components/rosette/TooltipPopup.tsx

```diff
@@ -0,0 +1,72 @@
+import cx from 'classnames'
+import React from 'react'
+
+import { MediumRosette } from './Rosette'
+import { RiskSentiments, RiskValue, RiskValues } from './types'
+
+export interface RosetteTooltipProps {
+  riskSummary: RiskSentiments
+  riskValues: RiskValues
+}
+
+export function RosetteTooltipPopup({
+  riskValues,
+  riskSummary,
+}: RosetteTooltipProps) {
+  return (
+    <div className="w-88 flex flex-col">
```

Comment:
this w-88 caught my eye, why such an arbitrary value?

Reply:
Good catch, this class doesn't even exist